build.rs

```diff
@@ -27,4 +27,44 @@ fn main() {
     }
     let target = env::var("TARGET").unwrap();
     println!("cargo:rustc-env=TARGET={target}");
+
+    // Set linker options specific to Windows MSVC.
+    let target_os = env::var("CARGO_CFG_TARGET_OS");
+    let target_env = env::var("CARGO_CFG_TARGET_ENV");
+    if target_os.as_deref() != Ok("windows") && target_env.as_deref() != Ok("msvc") {
```

Comment:
The De Morgan's Law is being dangerous here: looking at L31, I think you meant:

```suggestion
    if !(target_os.as_deref() == Ok("windows") && target_env.as_deref() == Ok("msvc")) {
```

I could've used `||` but it's less clear.

Reply:
Oh yes. Refactoring is dangerous 🙂.

Reply:
Nice catch!

Reply:
"Given enough eyeballs, all bugs are shallow" :)